### ISSUE TITLE
AFHDS support is missing in internal RF menu. 

### ISSUE THREAD

**Issue by rakesh-i** [OpenI6X/opentx#150]
Stock firmware supports older AFHDS receivers but not able to select the AFHDS mode in internal RF menu. Only supports AFHDS2A receivers.(version 1.3.0)

**Comment by marianomd** [comment]
I'm working on this. #168

**Comment by rakesh-i** [comment]
Thank you! So much. I few older receivers laying around.

**Comment by Dronard** [comment]
Hello,
I use openi6x_1.5.0_fr.bin, and I have only FS2A mode or OFF, how to get AFHDS?

Thank for reply

**Comment by rakesh-i** [comment]
Implementation is in progress. Check project>TODO>in progress

**Comment by Dronard** [comment]
But It's write since 1.4.0 AFHDS is compatible... but not on 1.5.0?

**Comment by rakesh-i** [comment]
I have 1.4.0 it doesn't support afhds. What build you are using?

**Comment by Dronard** [comment]
![20220512_203403](https://user-images.githubusercontent.com/98915844/168145004-9d1576d0-1cfb-47e3-b369-fa6cb196010a.jpg)

I want to use this

**Comment by rakesh-i** [comment]
> ![20220512_203403](https://user-images.githubusercontent.com/98915844/168145004-9d1576d0-1cfb-47e3-b369-fa6cb196010a.jpg)
> 
> I want to use this

Yes me too. That's why I opened this thread. marianomd is trying to implement it. I'm waiting for it to complete. Check projects tab for progress

**Comment by Dronard** [comment]
Okay!
I don't go often on github, If you can reply here a message like AFHD is ready, I will thank you))

**Comment by rakesh-i** [comment]
> Okay! I don't go often on github, If you can reply here a message like AFHD is ready, I will thank you))

Okay

**Comment by Dronard** [comment]
Hello ! Does the 1.6.1 version support AFHDS?

**Comment by rakesh-i** [comment]
Haven't tried it yet. Most probably not

**Comment by marianomd** [comment]
Not yet.

**Comment by Dronard** [comment]
Hello!
Any news about it? should we expect to get it on the 1.7.0 ?

**Comment by Dronard** [comment]
is it solve?

**Comment by Dronard** [comment]
so? have you got an idea of... when?

**Comment by marianomd** [comment]
I have no time for this, sorry. If someone is willing to take over it, it's here: #168

**Comment by Dronard** [comment]
1.8.0 Millésime is here 👌🏻👍🏻

**Comment by Dronard** [comment]
I install 1.8.0 version... I don't have it, could you explain how you do it work?

**Comment by Dronard** [comment]
i don't have AFHDS2 mode...

**Comment by Dronard** [comment]
![20221109_144652](https://user-images.githubusercontent.com/98915844/200846769-717918d3-e8eb-49bb-8680-b856a1149571.jpg)
![20221109_144546](https://user-images.githubusercontent.com/98915844/200846779-bf533057-fe57-4992-a108-569c6021cc39.jpg)
![20221109_144539](https://user-images.githubusercontent.com/98915844/200846783-222ed942-3f39-4ec4-beec-2889265bb5f5.jpg)

Here is what I have in RF binding menu : FS2A or OFF

**Comment by Dronard** [comment]
😥 please...

**Comment by Dronard** [comment]
oh, i'm wrong i'm wrong, I hope there will be a new firmware which will get afhds compatibility, in addition to afhds2

**Comment by Dronard** [comment]
Hello! any news about it?
